Makefile

```diff
@@ -230,9 +233,10 @@ web3_dir           := $(abspath $(DEFN_DIR)/web3)
 web3_files         := $(ALL_FILES)
 web3_main_module   := WEB3
 web3_syntax_module := $(web3_main_module)
-web3_main_file     := web3
+web3_main_file     := web3.md
+web3_main_filename := $(basename $(notdir $(web3_main_file)))
 web3_kompiled      := $(web3_dir)/build/kevm-client
-web3_kore          := $(web3_dir)/$(web3_main_file)-kompiled/definition.kore
+web3_kore          := $(web3_dir)/$(web3_main_filename)-kompiled/definition.kore
 export web3_main_file
```

Comment:
This build seems to fail due to this.  @ehildenb Do you know what I should do for this export?

Reply:
nvm